plugins/modules/vmware_guest_info.py

```diff
@@ -74,11 +74,21 @@
    tags:
      description:
      - Whether to show tags or not.
-     - If set C(True), shows tag information.
+     - If set C(True), shows tags information.
      - If set C(False), hides tags information.
-     - vSphere Automation SDK and vCloud Suite SDK is required.
+     - vSphere Automation SDK is required.
+     - 'tags' returns a list of tag names.
```

Comment:
```suggestion
     - "'tags' returns a list of tag names."
```

The following error occurs by the cause is this line.

```
ERROR: Command "ansible-doc -t module community.vmware.vmware_guest_info" returned exit status 1.
>>> Standard Error
ERROR! module community.vmware.vmware_guest_info missing documentation (or could not parse documentation): while parsing a block mapping
  in "<unicode string>", line 64, column 6:
         description:
         ^
expected <block end>, but found '<scalar>'
  in "<unicode string>", line 69, column 15:
         - 'tags' returns a list of tag names.
                  ^
```